Sources/macros.js

```diff
@@ -763,22 +755,13 @@ export function algo(publicAPI, model, numberOfInputs, numberOfOutputs) {
     count = model.numberOfInputs;
     while (count--) {
       if (
-        model.inputConnection[count] &&
-        model.inputConnection[count].filter.shouldUpdate()
+        model.inputConnection[count]?.filter.shouldUpdate() ||
```

Comment:
@floryst @jourdain are we OK with optional chaining in VTK.JS ?

Reply:
I thought we are based on the latest update of the toolchain. Am I correct @floryst ?

Reply:
Yep, we handle optional chaining (and null coalescing, for that matter).

Reply:
Good to know, thanks !